integration/mesos/src/main/java/tachyon/mesos/TachyonFramework.java

```diff
@@ -48,9 +48,11 @@
     private String mMasterHostname = "";
     private String mTaskName = "";
     private int mMasterTaskId;
+    private int mWorkerTaskId;
```

Comment:
what does `mWorkerTaskId` mean here? Maybe my understanding is wrong, aren't we expected to have multiple worker tasks? Shouldn't they have different Ids?